admin/src/Hooks/useAlert.js

```
@@ -22,6 +23,24 @@ export function useAlert() {
     title,
   }) {
+    const docsLabel = i18n(
+      'plugin.message.error.meilisearchDocsLink',
+      'See more',
+    )
+
+    let normalizedLink = link
+    if (link && link.label && typeof link.label === 'object') {
+      const { id, defaultMessage } = link.label
+      const normalizedId =
+        id?.replace(new RegExp(`^${pluginId}\\.`), '') || id
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)